manifests/plugin/abrt.pp

```diff
@@ -30,7 +33,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $group                   = $::foreman_proxy::plugin::abrt::params::group,
+  $version                 = $::foreman_proxy::plugin::version,
```

Comment:
But what will be the default? Should I add a pick:

    $version_real = pick($version, 'installed')
    validate_string($version_real)